src/UITests/UILibrary/Settings.cs

```diff
@@ -0,0 +1,38 @@
+﻿using OpenQA.Selenium.Appium.Windows;
+
+namespace UITests.UILibrary
+{
+    public class Settings
+    {
+        WindowsDriver<WindowsElement> Session;
+        ApplicationTab ApplicationTab { get; }
+        AboutTab AboutTab { get; }
+
+        public Settings(WindowsDriver<WindowsElement> session)
+        {
+            Session = session;
+            AboutTab = new AboutTab(session);
+            ApplicationTab = new ApplicationTab(session);
+        }
+        public bool SaveAndClose() => false;
+        public bool Back() => false;
+    }
+
+    public class ApplicationTab
```

Comment:
I am aware that a few of these files contain multiple classes per file. I'm pretty split. 
On one hand, the only reason they're in multiple classes is so that when someone uses the library, they get to see the distinction (ie `Settings.ApplicationTab.<application tab specific function>` instead of `Settings.<application tab specific function>`. So in that sense, I do think all of their eventual methods do make sense in one file. 
On the other hand, multiple classes in one file is definitely not the standard. 

I'm curious if anyone agrees with my first point or if these should just be split into multiple files.

Reply:
I generally prefer to have the class name and the file name be identical, but this is an unusual case. Maybe we can put them in a single file for now, then refactor as needed later? As an aside, since "ApplicationTab" is explicitly in the context of settings, should we consider naming the class SettingsApplicationTab? We could also use a namespace, but that feels a little heavy

---
In reply to: [299116031](https://github.com/microsoft/accessibility-insights-windows/pull/408#discussion_r299116031) [](ancestors = 299116031)

Reply:
Sounds good. We can revisit the ApplicationTab name when we add the Settings tests.